Header header
int8 points
float64 lat1
float64 lat2
float64 lat3
float64 lat4
float64 lat5
float64 lat6
float64 lat7
float64 lat8
float64 lon1
float64 lon2
float64 lon3
float64 lon4
float64 lon5
float64 lon6
float64 lon7
float64 lon8
float32 alt1
float32 alt2
float32 alt3
float32 alt4
float32 alt5
float32 alt6
float32 alt7
float32 alt8
